### DISCUSSION TITLE
limit_nonpersistent_absolute wird nicht übernommen

### DISCUSSION THREAD

**Discussion by Chilled** [tbnobody/OpenDTU#748] [Q&A] [answered]
Hallo zusammen.

Ich versuche aktuell eine Nulleinspeisung in Verbindung mit IP-Symcon umzusetzen. Ich nutze in Symcon das Modul [https://github.com/roastedelectrons/HoymilesOpenDTU](https://github.com/roastedelectrons/HoymilesOpenDTU) . 
Ich habe auch schon versucht händisch ein Limit zu setzen. Im Broker (IP-Symcon) wird der Wert auch gesetzt (notretained) unter solar/serial/cmd/limit_nonpersistent_absolute
<img width="462" alt="Bildschirm­foto 2023-03-25 um 16 50 04" src="https://user-images.githubusercontent.com/3774225/227727901-c9cfde1e-853d-4c18-b04f-6a9e1cf8b36e.png">

Allerdings übernimmt die openDTU die Werte nie. Auch als persistent funktioniert das ganze nicht. Hat jemand einen Rat? In der Webinterface Console bekomme ich auch keine MQTT Meldungen

**Comment by zfrank2601** [comment]


**Reply by Chilled** [reply]
Absolute ist doch Watt laut Doku oder habe ich jetzt 10 mal falsch gelesen? relative sollen % sein

**Reply by zfrank2601** [reply]
sorry, stimmt, war noch zu früh heute :-(
Dann sollt es aber funktionieren

**Comment by Chilled** [comment, marked as answer]
Ok.... hat sich erledigt. Ein Reset der DTU brachte Erfolg. Jetzt funktioniert alles wie gewünscht. Trotzdem danke
